### ISSUE TITLE
Use my own data to finetune a model to predict?

### ISSUE THREAD

**Issue by aureosun** [kwotsin/transfer_learning_tutorial#12]
Hello,I'm a beginner of deep learning,I have fintued inceptionv3 on my own data,now I'd like to use the finetuned model to predict,what should I do? It means that the input is .jpg image,not the .tfrecord files,and still with no labels attach to it(compared with .tfrecord files) ,and output is the prediction of the class that the image belongs to .Could you give me some suggestions?

**Comment by kwotsin** [comment]
Do you mean making your model suitable for performing inferences? For a start, you can try to freeze your model with this code: https://github.com/kwotsin/TensorFlow-Xception/blob/master/write_pb.py to produce a pb file that is more lightweight.

Then you can try to feed in one image into the input node of the pb graph like this:

```
import tensorflow as tf
from scipy.misc import imread
import numpy as np

height, width = 299, 299
image_filename = "my_image.jpg"
image = imread(image)
image = np.resize(image, (height, width, 3))
input_node_name = 'my_input_node'
output_node_name = 'my_output_node'

graph_filename = "my_graph.pb"
with tf.gfile.GFile(graph_filename, 'rb') as f:
    graph_def = tf.GraphDef()
    graph_def = ParseFromString(f.read())

with tf.Graph().as_default() as graph:
    tf.import_graph_def(graph_def)
    input_node = graph.get_tensor_by_name(input_node_name)
    output_node = graph.get_tensor_by_name(output_node_name)

    with tf.Session() as sess:
        prediction = sess.run(output_node, feed_dict={input_node: image})
```

To check for the input node names, run in the default graph context:

```
for op in graph.get_operations():
    print op.values()
```
